scripts/test.mk

```diff
@@ -56,11 +80,18 @@ _test:
 
 .SILENT: \
 	_test \
+	test \
```

Comment:
As discussed, could we check if the test types listed here match those in the SQEF, please? Here are some suggestions:
- test-accessibility - keep as is
- test-breakpoint - rename it to `capacity`
- test-code-quality - remove, as we already have SonarCloud
- test-contract (API) - keep as is
- test-coverage - keep it as it is a separate concern to unit test focusing specifically on the test coverage
- test-endurance - rename it to `soak`
- test-integration - keep as is
- test-lint
- test-load - this is a convenience wrapper
- test-response-time - this is a baseline test / response time test
- test-security
- test-ui - regression test, keep it as is
- test-ui-performance - this tests the render behaviour of a browser, aka lighthouse test
- test-unit

As an option we could also reordering these to match the 'test pyramid' order (bottom-up).